### PR TITLE
add dark mode breadcrumbs

### PR DESCRIPTION
**Pull request by alexusernamebruh**

### CODE REVIEW

File: src/components/Breadcrumbs/index.tsx [hunk 1]
```diff
@@ -25,17 +25,17 @@ export const Breadcrumbs: React.FC<BreadcrumbsProps> = ({
   },
 }) => {
   return (
-    <nav className='flex' aria-label='Breadcrumb'>
+    <nav className='flex ' aria-label='Breadcrumb'>
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.tsx]: remove the space

**Review by RichardX366 — CHANGES_REQUESTED** [review]
Can you do the `darkMode: 'class'` inside the tailwind config thing? https://tailwindcss.com/docs/dark-mode

**Comment by RichardX366** [comment]
Also attach screenshot of dark and not dark

**Comment by alexusernamebruh** [comment]
Kk, will do tonight

**Comment by mxdi9i7** [comment]
can we also add a prop to the component that allows dark to be set manually by the user?

**Comment by mxdi9i7** [comment]
a prop called `dark` that is a boolean

**Comment by alexusernamebruh** [comment]
yes sir

File: src/components/Breadcrumbs/index.tsx [hunk 2]
```diff
@@ -23,52 +24,153 @@ export const Breadcrumbs: React.FC<BreadcrumbsProps> = ({
...
-          </div>
-        </li>
-        {links.slice(1).map((page, i) => (
-          <li key={i} title={page.title}>
-            <div className='flex items-center'>
-              <ChevronRightIcon
-                className='flex-shrink-0 w-5 h-5 text-gray-300'
-                aria-hidden='true'
-              />
+  if (darkMode === true) {
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.tsx]: dont change the whole component just modify the classes

File: src/components/Breadcrumbs/index.stories.tsx [hunk 3]
```diff
@@ -26,10 +26,22 @@ const props: BreadcrumbsProps = {
...
 };
-
+let className = '';
+if (props.darkMode === false) {
+  className =
+    'px-2 py-2 border border-gray-300 rounded-lg hover:border-gray-600';
+}
+if (props.darkMode === true) {
+  className =
+    'bg-gray-900 px-2 py-2 border border-gray-300 hover:border-gray-600 rounded-lg ';
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.stories.tsx]: remove ending space

File: src/components/Breadcrumbs/index.stories.tsx [hunk 4]
```diff
@@ -26,10 +26,22 @@ const props: BreadcrumbsProps = {
     },
   ],
 };
-
+let className = '';
+if (props.darkMode === false) {
```
Comment (RichardX366) [inline src/components/Breadcrumbs/index.stories.tsx]: use else ifs.
put in order
props.darkMode
props.darkMode === false
else

**Review by RichardX366 — CHANGES_REQUESTED** [review]

**Pull request merged**